fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java

```diff
@@ -247,6 +222,67 @@ public Connection getConnection() throws FHIRPersistenceDBConnectException {
         }
     }
 
+    /**
+     * this feature is executes row access control
+     * 
+     * @param connection
+     * @throws Exception
+     */
+    public void executeRowAccessFeature(Connection connection) throws Exception {
+        // For the multi-tenant feature, configure the connection for the tenant by setting the sv_tenant_id.
+        String tenantName = FHIRRequestContext.get().getTenantId();
+        String tenantKey = null;
+        String dsId = FHIRRequestContext.get().getDataStoreId();
+        boolean multiTenantFeature = false;
+
+        // Retrieve the property group pertaining to the specified datastore.
+        // Find and set the tenantKey for the request, otherwise subsequent pulls from the pool
+        // miss the tenantKey.
+        String dsPropertyName = FHIRConfiguration.PROPERTY_DATASOURCES + "/" + dsId;
+        PropertyGroup dsPG = FHIRConfigHelper.getPropertyGroup(dsPropertyName);
+        if (dsPG != null) {
```

Comment:
should we handle dsPG == null situation?

Reply:
let me add a log

Reply:
and maybe a exception?

Reply:
fixed, it's a good idea

Reply:
cool, thanks!